javascripts/helpers.js

```diff
@@ -0,0 +1,46 @@
+// allows us to compare an array of arrays
+// by turning the inner arrays into string, e.g. ["r3, c3", "r2, c0"]
```

Comment:
This comment is a little misleading because there is no space in the returned strings. I would update the comment with a more representative return value:
```javascript
["r3,c3", "r2,c0"]
```